src/main/java/com/tannat/country/controllers/PageController.java

```diff
@@ -0,0 +1,35 @@
+package com.tannat.country.controllers;
+
+import com.tannat.country.dtos.CityDto;
+import com.tannat.country.dtos.CountryDto;
+import com.tannat.country.services.CityService;
+import com.tannat.country.services.CountryService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/pages")
+@RequiredArgsConstructor
+public class PageController {
```

Comment:
It is not required to add a separate controller for pagination. add optional request parameters variables to the get requests

Reply:
ok